### ISSUE TITLE
Crash starting tests on XCode 8

### ISSUE THREAD

**Issue by jznadams** [kif-framework/KIF#921]
Since updating to Xcode 8, I've had all kinds of problems getting my test cases to run. Trying to get back to a baseline, I just deleted my entire test target and re-created it from scratch using the instructions in the readme here. It's definitely a unit test target.  I removed the KIF cocoapod and added back my basic unit tests and they run fine.

My next step is getting the KIF tests back in there, so I have added the pod and without touching anything else. Now, I see the following exception every time I try to run my unit tests on the iOS 10 simulator:

```

2016-10-21 16:36:22.527 TI[18326:7453239] -[AccessibilitySettingsController AXInspectorEnabled:]: unrecognized selector sent to instance 0x7f95ac053a00
2016-10-21 16:36:29.570 TI[18326:7453239] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[AccessibilitySettingsController AXInspectorEnabled:]: unrecognized selector sent to instance 0x7f95ac053a00'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010f03734b __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x000000010e0bc21e objc_exception_throw + 48
	2   CoreFoundation                      0x000000010f0a6f34 -[NSObject(NSObject) doesNotRecognizeSelector:] + 132
	3   CoreFoundation                      0x000000010efbcc15 ___forwarding___ + 1013
	4   CoreFoundation                      0x000000010efbc798 _CF_forwarding_prep_0 + 120
	5   Tests                               0x00000001216d26a9 -[KIFAccessibilityEnabler _enableAccessibility] + 557
	6   XCTest                              0x0000000120a41f5e -[XCTestObservationCenter _observeTestExecutionForBlock:] + 501
	7   XCTest                              0x0000000120a209a9 -[XCTestDriver _runSuite] + 436
	8   XCTest                              0x0000000120a217a2 -[XCTestDriver _checkForTestManager] + 287
	9   XCTest                              0x0000000120a70f5c _XCTestMain + 628
	10  CoreFoundation                      0x000000010efdc25c __CFRUNLOOP_IS_CALLING_OUT_TO_A_BLOCK__ + 12
	11  CoreFoundation                      0x000000010efc1304 __CFRunLoopDoBlocks + 356
	12  CoreFoundation                      0x000000010efc0a75 __CFRunLoopRun + 901
	13  CoreFoundation                      0x000000010efc0494 CFRunLoopRunSpecific + 420
	14  GraphicsServices                    0x0000000111c54a6f GSEventRunModal + 161
	15  UIKit                               0x000000010bf5ff34 UIApplicationMain + 159
	16  TI                                  0x0000000108dbca5f main + 111
	17  libdyld.dylib                       0x00000001106c968d start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```

I'm using the head of the master branch right now to make sure I have all of the latest changes, and my pod file looks like this:

```
platform :ios, '8.0'

target 'TI' do
  ...
end

target 'Tests' do
  pod 'KIF', :git => 'https://github.com/kif-framework/KIF.git', :configurations => ['Debug', 'Debug Local Server', 'Debug Staging Server']
end
```

Any ideas? It doesn't even seem to be getting to my test cases, and trying to switch various combinations in and out doesn't seem to make a difference.

**Comment by jznadams** [comment]
I did manage to get things working, but it is not entirely clear to me why. I have several different server environments, so I have several different debug build configurations that point towards the different environments. The regular "Debug" configuration points to production for me, so I rarely use it in test cases. All of the testing I have been doing up until now has been using a different configuration, "Debug Staging Server". 

On a whim, I tried the regular debug configuration and found that the crash I listed above doesn't happen. My tests fail since it's pointing towards the wrong environment, but it seems to get around the problem. I've been using KIF successfully with multiple debug configurations for some time now, the only thing that changed was starting to use Xcode 8. 

I can make this work for now. I've deleted the other debug build configurations for the time being, and I might re-create them from scratch and see if I run into that crash again. I'm happy with closing this issue if you don't see a reason why KIF should have issues running on multiple build configurations.
